core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -125,13 +125,13 @@ describe('Creator dashboard controller', () => {
     });
   }));
 
-  it('should get exploration url successfully', function() {
+  it('should get exploration url when providing an exploration id', function() {
```

Comment:
Here and elsewhere: "it" refers to the function being tested. It isn't providing anything. So perhaps rephrase to "when provided with an exploration id". Ditto elsewhere.

Alternatively, if the condition is that "user does something" then say that, making the actor explicit.

Reply:
Done.